pyperformance/data-files/benchmarks/bm_tomllib_loads/generate_data.py

```diff
@@ -0,0 +1,22 @@
+from urllib.request import urlopen
+import json
+import toml
+
+BASE_URL = "https://api.github.com/repos/python/cpython/pulls?per_page=1000&state=all"
+
+def main():
+    page = 1
+    all_issues = []
+    while page <= 10:
+        with urlopen(f"{BASE_URL}&page={page}") as response:
+            issues = json.loads(response.read())
+            if not issues:
+                break
+            all_issues.extend(issues)
+            print(f"Page: {page} Total Issues: {len(all_issues)}")
+        page += 1
```

Comment:
FWIW, this is often spelled as:
```suggestion
    all_issues = []
    for page in range(1, 11):
        with urlopen(f"{BASE_URL}&page={page}") as response:
            issues = json.loads(response.read())
            if not issues:
                break
            all_issues.extend(issues)
            print(f"Page: {page} Total Issues: {len(all_issues)}")
```

Reply:
done